client/reader/controller.js

```diff
@@ -24,6 +26,11 @@ import { preload } from 'sections-preload';
 import { renderWithReduxStore } from 'lib/react-helpers';
 import AsyncLoad from 'components/async-load';
 
+// these three are included to ensure that the stores required have been loaded and can accept actions
+import FeedSubscriptionStore from 'lib/reader-feed-subscriptions'; // eslint-disable-line no-unused-vars
+import PostEmailSubscriptionStore from 'lib/reader-post-email-subscriptions'; // eslint-disable-line no-unused-vars
+import CommentEmailSubscriptionStore from 'lib/reader-comment-email-subscriptions'; // eslint-disable-line no-unused-vars
```

Comment:
I hoisted these three imports to the top in order to replace all use of `require`. However, I'm not sure if hoisting these will break the intended side effects the `require`s intended when they were inside the `loadSubscriptions` function.